packages/db-states-redis/src/lib/states/statesInRedisClient.ts

```diff
@@ -163,10 +226,11 @@ class StateRedisClient {
             );
         } else if (Array.isArray(this.settings.connection.host)) {
             // Host is an array means we use a sentinel
-            const defaultPort = Array.isArray(this.settings.connection.port) ? null : this.settings.connection.port;
             this.settings.connection.options.sentinels = this.settings.connection.host.map((redisNode, idx) => ({
                 host: redisNode,
-                port: defaultPort || this.settings.connection.port[idx]
+                port: Array.isArray(this.settings.connection.port)
+                    ? this.settings.connection.port[idx]
+                    : this.settings.connection.port
```

Comment:
This looks like it could be extracted to a helper function ;)

Reply:
Ah yes that was the one where I forgot a todo 👍🏼